util/fqdn.go

```diff
@@ -7,15 +7,15 @@ import (
 	"strings"
 )
 
-func removeQuestionMarks(domain string) string {
+func RemoveQuestionMarks(domain string) string {
```

Comment:
This doesn't look like this handles multiple question marks. I think `?.?.domain` is an acceptable redacted certificate.

Reply:
it does - it's in a for loop.

Reply:
Then it should probably be named `removePrependedQuestionMark`. The function name implies that it removes multiple and that's not the case.